hyperspy/hspy.py

```diff
@@ -53,6 +53,26 @@ def create_model(signal, *args, **kwargs):
         If None it will use the Hartree-Slater GOS if
         they are available, otherwise it will use the hydrogenic GOS.
 
+    If the signal is an EDS signal the following extra parameters
+    are available:
+
+    auto_add_lines : boolean
+        If True, and if spectrum is an EDS instance, it will
+        automatically add Gaussians for all X-rays generated in the energy
+        range by an element.
+    auto_background : boolean
+        If True, and if spectrum is an EDS instance adds automatically
+        a polynomial order 6 to the model.
+
+    Examples
+    --------
+
+    With s an EDS instance
+    >>> m = create_model(s)
+    >>> m.mulifit()
+    >>> m.plot()
+    >>> m.get_lines_intensity(plot_result=True)
+
```

Comment:
Yeah it was just there from Pierre's earlier stuff so I just left it, but I'll delete it so that the diff is smaller since it's about to make no difference...